packages/cli/templates/webcomponents/igc-ts/projects/side-nav/files/index.html

```diff
@@ -17,7 +17,7 @@
       padding: 25px;
```

Comment:
`:host` ?! What would that be in this setting?

Reply:
Also, could we not use an actual css file for this since it's not a component, but the index.html file?